src/components/layout/CardHeader.tsx

```diff
@@ -60,12 +60,9 @@ const CardHeader = function CardHeader({
       )}
       ref={downcastRef(elementRef)}
     >
-      {title && (
-        <CardTitle classes="grow" variant={variant}>
-          {title}
-        </CardTitle>
-      )}
+      {title && <CardTitle variant={variant}>{title}</CardTitle>}
       {children}
+      <div className="grow" />
```

Comment:
This isn't my favorite solution (empty "spacer" divs), but it can be helpful in situations where otherwise the `flex-grow` would need to be assigned to a different element depending on context. This is a little more foolproof.